google/ggrc-core
# GGRC-3931:  add components for Workflow ACL person on New/Edit modal

# Issue description
In scope of moving Workflow roles to ACL add components for Workflow people on Create/Edit modal.

# Steps to test the changes
Due to absence of back-end support for Workflow ACL we need to test changes in the provided way
1) Click on Start New Workflow
2) In Web Tools Console add breakpoint in workflow.js at the begining of init()
3) run the  following lines into console 
```javascript
GGRC.access_control_roles.push({
default_to_current_user:false,
delete:true,
id:85,
mandatory:false,
modified_by:null,
my_work:false,
name:"Workflow Member",
non_editable:false,
object_type:"Workflow",
read:true,
selfLink:"/api/access_control_roles/83",
tooltip:null,
type:"AccessControlRole",
update:true});

GGRC.access_control_roles.push({
default_to_current_user:true,
delete:true,
id:83,
mandatory:true,
modified_by:null,
my_work:false,
name:"Admin",
non_editable:false,
object_type:"Workflow",
read:true,
selfLink:"/api/access_control_roles/83",
tooltip:null,
type:"AccessControlRole",
update:true})
```
4) Try to save Workflow and check request in Network tab.

# Solution description
Reuse existing components for custom roles and other components, update modal style and remove redundant code in workflow.js.

# Sanity checklist


# PR Review checklist
